tests/test_edgeql_syntax.py

```diff
@@ -1455,7 +1455,7 @@ def test_edgeql_syntax_shape_43(self):
         """
 
     @tb.must_fail(errors.EdgeQLSyntaxError,
-                  r"Unexpected 'foo'", line=3, col=27)
+                  r"Missing ',' before the shape item 'foo'", line=3, col=27)
```

Comment:
I don't think the definite article should be used in any of these messages, because, especially in the case of tuples and arrays, there may be multiple items `'foo'`.

Reply:
I've reworded the error messages.